README.md

```diff
@@ -1,51 +1,71 @@
 # ssb-ebt
 
-Adapter for [epidemic-broadcast-trees](https://github.com/dominictarr/epidemic-broadcast-trees)
-to [secure-scuttlebutt](http://scuttlebutt.nz/)
+*Replicates SSB feeds using the efficient "Epidemic broadcast tree" algorithm.*
 
+This module is an adapter for the module [epidemic-broadcast-trees](https://github.com/ssbc/epidemic-broadcast-trees)
+to work with [secure-scuttlebutt](http://scuttlebutt.nz/). Supersedes
+[ssb-replicate](https://github.com/ssbc/ssb-replicate).
 
-## testing/debugging
+## Installation
 
-There are several scripts in `./debug` which I used for testing
-ebt replicaiton.
+**Prerequisites:**
 
-use `./debug/remote.js <address>` to connect to an sbot running
-ebt and replicate. Running this won't store anything locally,
-it will just download everything drop it on the floor. This
-is used to test performance of ebt on a server.
+- Requires **Node.js 10** or higher
+- Requires **ssb-db** or **ssb-db2**
 
-I normally see values between 2k and 3k messages per second,
-that replicates 100k messages in under a minute.
+```
+npm install --save ssb-ebt
+```
+
+Add this secret-stack plugin like this:
+
+```diff
+ const SecretStack = require('secret-stack')
+ const caps = require('ssb-caps')
+
+ const createSsbServer = SecretStack({ caps })
+     .use(require('ssb-master'))
+     .use(require('ssb-db'))
++    .use(require('ssb-ebt'))
+     .use(require('ssb-friends'))
+     // ...
+```
+
+## Usage
+
+ssb-ebt itself does **NOT** trigger replication automatically after it's
+installed, instead, you need to call its API methods yourself (primarily
+`request` or `block`), or use a scheduler module such as
+[ssb-replication-scheduler](https://github.com/ssb-ngi-pointer/ssb-replication-scheduler).
 
-## api
+### `ssb.ebt.request(destination, replicating)`
 
-### ebt.block (from, to, blocking)
+Request that the SSB feed ID `destination` be replicated. `replication` is a
+boolean, where `true` indicates we want to replicate the destination. If set to
+`false`, replication is stopped.
```

Comment:
Is this a sync method. Does it return nothing?

Reply:
Fixed, thanks!